<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectBrowser</class>
 <widget class="QDockWidget" name="ObjectBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>path</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="text">
         <string>browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Id: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="idLine"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>X: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xLine"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Y: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yLine"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>langle: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="laLine"/>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>rangle: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="raLine"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="isThrowable">
           <property name="text">
            <string>Is Throwable</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListView" name="objects"/>
         </item>
         <item>
          <widget class="QListView" name="neighbors"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
